librad/src/meta/entity.rs

```diff
@@ -142,46 +142,18 @@ pub enum VerificationStatus {
 }
 
 impl VerificationStatus {
-    pub fn verified(&self) -> bool {
-        if let VerificationStatus::Verified = self {
-            true
+    pub fn verification_failed(&self) -> Option<&Error> {
```

Comment:
Could we preserve the success cases as well (a la `is_ok()`)? Perhaps `VerificationStatus` could actually be `Result<SuccessCases, ErrorCases>`

Reply:
Well, my aim here was to use the success case directly when someone wants to compare or match on. I think https://github.com/radicle-dev/radicle-link/issues/121 could make any of this obsolete too. I imagine we'd have `Result<Entity<Signed, T>, Error>`. Does that sound right to you?